app/pages/home-common/social.jsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import Translate from 'react-translate-component';
+import { Link } from 'react-router';
+import counterpart from 'counterpart';
+import moment from 'moment';
+import ProjectCard from '../../partials/project-card';
+import { getPublication, getRecentProjects, getBlogPosts, getNewestProject } from '../../lib/get-social-data';
+
+counterpart.registerTranslations('en', {
+  socialHomePage: {
+    community: 'The Zooniverse Community',
+    daily: 'The Daily Zooniverse',
+    news: 'News',
+    newestProject: 'Newest Project',
+    recentProjects: 'Recent Project Updates',
+    recentPublications: 'Recent Publications'
+  }
+});
+
+export default class HomePageSocial extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      blogPosts: [],
+      newestProject: {},
+      newestPublication: {},
+      recentProjects: []
+    };
+  }
+
+  componentDidMount() {
+    this.getSocial();
+  }
+
+  getSocial() {
+    getNewestProject().then((newestProject) => {
+      this.setState({
+        newestProject,
+        newestPublication: getPublication()
```

Comment:
Why does this wait to get the newest project before setting the newest publication? I can't see anything about publications that needs to wait for the project.